feature-libs/commerce-quotes/core/facade/commerce-quotes.service.ts

```diff
@@ -39,6 +53,95 @@ export class CommerceQuotesService implements CommerceQuotesFacade {
       { reloadOn: [CommerceQuotesListReloadQueryEvent] }
     );
 
+  protected createQuoteCommand: Command<
+    { quoteMetadata: QuoteMetadata; quoteComment: Comment },
+    Quote
+  > = this.commandService.create<
+    { quoteMetadata: QuoteMetadata; quoteComment: Comment },
+    Quote
+  >(
+    (payload) =>
+      combineLatest([
+        this.userIdService.takeUserId(),
+        this.activeCartFacade.takeActiveCartId(),
+      ]).pipe(
```

Comment:
Maybe it's worth to ask e.g. Nik if it's possible to simplify this complex pipe somehow.